### PR TITLE
Updated for InterPro 101.0

### PR DESCRIPTION
**Pull request by blazaropinto**
Updates

### CODE REVIEW

File: docs/images/overview.png
Comment (typhainepl) [inline docs/images/overview.png:1]: The new screenshot has some black background towards the bottom, as the two images grouped together don't have the same width. Can you try to fix it?

**Review by typhainepl — CHANGES_REQUESTED** [review]
Good job, I made a few edits, but nothing major. One of the images isn't quite right.

File: docs/images/overview.png
Comment (blazaropinto) [reply docs/images/overview.png:1]: Thank you for noticing this, I missed it! That's it sorted :)

**Review by typhainepl — APPROVED** [review]
Ready to merge!

**Pull request merged**